src/routes.tsx

```diff
@@ -13,14 +13,46 @@ import React, { Suspense, useEffect } from "react";
 import { Navigate, Route, Routes } from "react-router-dom";
 
 // Import page components
-import { NotebookPage } from "@/pages/NotebookPage.tsx";
-import { NotebooksDashboardPage } from "@/pages/NotebooksDashboardPage.tsx";
-import OidcCallbackPage from "@/pages/OidcCallbackPage.tsx";
+
+// import {NotebookPage as NotebookPageComponent} from "@/pages/NotebookPage.tsx";
+const NotebookPage = React.lazy(() =>
+  import("@/pages/NotebookPage.tsx").then((mod) => ({
+    default: mod.NotebookPage,
+  }))
+);
+const NotebooksDashboardPage = React.lazy(() =>
+  import("@/pages/NotebooksDashboardPage.tsx").then((mod) => ({
+    default: mod.NotebooksDashboardPage,
+  }))
+);
+const OidcCallbackPage = React.lazy(() =>
+  import("@/pages/OidcCallbackPage.tsx").then((mod) => ({
+    default: mod.default,
+  }))
+);
+const ErrorFallbackPage = React.lazy(() =>
+  import("./components/ErrorFallbackPage").then((mod) => ({
+    default: mod.ErrorFallbackPage,
+  }))
+);
+const GeoJsonDemoPage = React.lazy(() =>
+  import("./pages/demo/geojson/GeoJsonDemoPage").then((mod) => ({
+    default: mod.GeoJsonDemoPage,
+  }))
+);
```

Comment:
In a future PR can you remove the GeoJson demo page and just have a notebook with the outputs you want to demo?

Reply:
I mean for your use of testing it out. Like, make a notebook with the GeoJSON instead of having additional paths on the deployed site or local app.